doc/src/guides/logo.rst

```diff
@@ -0,0 +1,32 @@
+===========
+SymPy Logo
+===========
+
+We would like to make it easy for you to include the SymPy project identity in
+your next academic paper, course materials, or presentation.
+
+.. image::  ../logo/sympy.svg
+  :width: 600
+  :align: center
+  :alt: SymPy Logo
+
+You can find this high-resolution version of the SymPy logo
+`here <https://github.com/sympy/sympy/blob/master/doc/src/logo/sympy.svg>`_.
```

Comment:
There's no reason to have this since the above image is basically the same thing.

Reply:
Removed.